### ISSUE TITLE
Provide way to define section/group header type and classes

### ISSUE THREAD

**Issue by j-4** [pcooksey/bibtex-js#31]
Currently, section/group headers are hard coded to start with `<H1>`. However, my homepage layout requires these headers to be `<H3>` or less to fit into the hierarchy.
Is it possible to provide a way to specify the header type as well as extra header styles?
It seems that these headers are actually the only (?) part of the framework, that cannot be specified by a html style.

Maybe instead of/in addition to specifying a `div` for every section, the header can be directly specified in the `bibtex_structure`?
```
<div class="bibtex_structure">
  <div class="sections bibtextypekey">
    <div class="section @article" title="Refereed Articles"></div>
  </div>
</div>
```
->
```
<div class="bibtex_structure">
  <div class="sections bibtextypekey">
    <h3 class="section @article custom-title-class">Refereed Articles</h3>
  </div>
</div>
```
It seems that you are able to get a header hierarchy of sections and groups, no idea how this would work with my suggestion above though.

**Comment by pcooksey** [comment]
Yes, I completely agree that it should be provided in the bibtex_structure. 
Fixed in commits: bf15038f740da194b16c090f5b1ae1feb72808cb and e88bdee59ff0ef099eb691ebc3f2e24f26e12909

I've updated the wiki pages to reflect the changes so it should all be up to date.
1. [Custom group headers](https://github.com/pcooksey/bibtex-js/wiki/Styles#custom-group-headers)
2. [Academic Style](https://github.com/pcooksey/bibtex-js/wiki/Styles#academic-style)

Note: there were some major changes to the academic style in order to have custom headers so you will have to change some html. For html code for the cost of flexibility.
